<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>377</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>377</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>377</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Новая марка</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="LE_number">
          <property name="placeholderText">
           <string>Номер марки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_country">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Страна</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_serial_number">
          <property name="placeholderText">
           <string>Номер серии</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="LE_serial_theme">
          <property name="placeholderText">
           <string>Тема серии</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_year">
          <property name="placeholderText">
           <string>Год</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_color">
          <property name="placeholderText">
           <string>Цвет</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="LE_size">
          <property name="placeholderText">
           <string>Размер</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_price">
          <property name="placeholderText">
           <string>Цена</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_stamp_theme">
          <property name="placeholderText">
           <string>Тема марки</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="PB_add">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="LE_del_theme">
        <property name="placeholderText">
         <string>Тема</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_del">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Удалить все</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLineEdit" name="LE_ch_number">
        <property name="placeholderText">
         <string>Номер марки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_ch_pos">
        <property name="placeholderText">
         <string>Новое место</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_ch_pos">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Изменить место</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="LE_section">
        <property name="placeholderText">
         <string>Раздел</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_sec_find_country">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Найти страны</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLineEdit" name="LE_cnt_theme">
        <property name="placeholderText">
         <string>Тема</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_th_country">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Найти страны</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="LE_serial">
        <property name="placeholderText">
         <string>Серия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_find_volume">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Найти том</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="LE_fnd_theme">
        <property name="placeholderText">
         <string>Тема</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_find_pos">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Найти места</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="LE_fnd_size">
        <property name="placeholderText">
         <string>Размер</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_find_theme">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Найти темы</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLineEdit" name="LE_fnd_pos">
        <property name="placeholderText">
         <string>Место</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_pos_find_country">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Найти страну</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="PB_report">
      <property name="text">
       <string>Получить отчет</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
